Sources/NIOCore/ChannelPipeline.swift

```diff
@@ -2089,3 +2091,171 @@ extension ChannelPipeline: CustomDebugStringConvertible {
         return handlers
     }
 }
+
+extension ChannelPipeline {
```

Comment:
These APIs will definitely want `syncOperations` versions.

Reply:
added in https://github.com/apple/swift-nio/pull/2918/commits/f5f85b531ca23799de9c80b87d174be296db31e3